src/main/java/Person.java

```diff
@@ -58,7 +60,19 @@ public Person getFather() {
     }
 
     public List<Person> getSiblings() {
-        return this.siblings;
+        return findSiblings();
+    }
+
+    private List<Person> findSiblings() {
+        var siblings = new HashSet<Person>();
+        if (mother != null) {
+            siblings.addAll(mother.getChildren());
+        }
+        if (father != null) {
+            siblings.addAll(father.getChildren());
+        }
+        siblings.remove(this); // remove this person because this person should ot be in the list of siblings
```

Comment:
Het commentaar in het Nederlands: Deze persoon is geen sibling van zichzelf, dus verwijder deze persoon uit de siblings lijst.